### PR TITLE
Fizz buzz python ds algo

### PR DESCRIPTION
**Pull request by shaunak56**
## Have you read the Contribution Guidelines?

Yes

## Description

Fizz Buzz is a typical interview question that tests the developer's knowledge of flow control and operators.

## Checklist
- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the README.md and link to my code.
 
## Related Issues (Mention Issue no. like #xxx)
#368 

Fixes Issue 
#368 
(Write your answer above.)

### CODE REVIEW

File: Python/FizzBuzz.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        elif i % 5 == 0:
+            print("Buzz")
+        # else print the number
+        else:
+            print(i)
+
+start_point = 1
+end_point = 100
+
+print_fizz_buzz(start_point, end_point)
```
Comment (sukritishah15) [inline Python/FizzBuzz.py:22]: Add below this Sample I/O, Time and Space Complexity in a multiline comment.

**Comment by sukritishah15** [comment]
@shaunak56 - README.md file changes are missing.

**Review by mephi007 — CHANGES_REQUESTED** [review]
Can you please modify Read.me file too, along with making changes to @sukritishah15's comment.

**Comment by shaunak56** [comment]
Sure. I will do the requested changes. Thank you.

**Pull request closed without merging**